src/components/autoSuggest/resultList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './resultList.css';
+
+const resultList = ({
+  entities,
+  entityKey,
+  uniqueKey,
+  redirectPath,
+  keyHeader,
+  keyValue,
+  i18Header,
+  i18Value,
+  entityIdxStart,
+  selectedIdx,
+  submitSearch,
+  selectedRowProps,
```

Comment:
It feels like`resultList` has too many params. They could be reduced by better splitting responsibility between this component and the `AutoSuggest` component. 
If `AutoSuggest` transofrms the results, then something like the following props could be enough:
```
{
header: { titleLeft, titleRight },
results: [ {id, valueLeft, valueRight, isSelected, type}, ... ],
onClick: (id, type) => {...},
}
```

Reply:
I like this idea, the only drawback I see is that there will be double iteration of the same object, first on the parent to prepare results based on isSelected, and to fill {id, valueLeft, valueRight}, then on the child to render those.